src/v2/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx

```diff
@@ -12,7 +12,7 @@ const SettingsEditProfileRoute: React.FC<SettingsEditProfileRouteProps> = ({
 }) => {
   return (
     <>
-      <Text>Edit Profile Route</Text>
+      <UserInformationRefetchContainer me={me} />
```

Comment:
I'm surprised this is a refetch container? Why is that?

Reply:
No idea, this was a recycled implementation that came along during the move. If you have a better solution/idea, I am open to it.